File: 1421-find-numbers-with-even-number-of-digits/1421-find-numbers-with-even-number-of-digits.py
class Solution:
    def findNumbers(self, nums: List[int]) -> int:
        count = 0
        for number in nums:
            digits = 0
            while number > 0:
                number //= 10
                digits += 1
            if digits % 2 == 0:
                count += 1
        return count
    

File: 1421-find-numbers-with-even-number-of-digits/1421-find-numbers-with-even-number-of-digits.java
class Solution {
    public int findNumbers(int[] nums) {
        int total_even = 0;
        for(int num: nums){
            int count = 0;

            while(num > 0){
                num /= 10;
                count++;
            }

            if(count % 2 == 0){
                total_even++;
            }
        }
        return total_even;
    }
}